components/validator/l2_output_submitter.go

```diff
@@ -393,14 +412,17 @@ func (l *L2OutputSubmitter) submitL2OutputTx(data []byte, nextBlockNumber *big.I
 		},
 	}
 
-	l.txCandidatesChan <- txmgr.TxCandidate{
+	txResponse := l.cfg.TxManager.SubmitTransaction(l.ctx, txmgr.TxCandidate{
 		TxData:     data,
 		To:         &l.cfg.L2OutputOracleAddr,
 		GasLimit:   0,
 		AccessList: accessList,
+	})
+	if txResponse.Err != nil {
+		l.log.Error("failed to send tx with response", "err", txResponse.Err)
+		return nil, txResponse.Err
 	}
-
-	return nil
+	return txResponse.Receipt, nil
```

Comment:
Accordingly, we can just return `l.cfg.TxManager.SubmitTransaction({ ... })`.
I think we don't need to log here.